.gitignore

```diff
@@ -0,0 +1,7 @@
+SOURCE_DOCUMENTS_/
```

Comment:
I am wondering why `.gitignore` is removed in the first place. 

Aside, I think you are adding extra files in your local environment and trying to ignore them here. Which should not be the case.